experiments/ClimaCore/Project.toml

```diff
@@ -1,6 +1,5 @@
 [deps]
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
-ClimaAtmos = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
```

Comment:
Was there a reason for these to be removed? I think we can revert these changes to this file.